src/service.rs

```diff
@@ -323,7 +323,7 @@ pub fn new_light(config: NodeConfiguration)
 			let fetch_checker = fetcher
 				.map(|fetcher| fetcher.checker().clone())
 				.ok_or_else(|| "Trying to start light import queue without active fetch checker")?;
-			let grandpa_block_import = sc_finality_grandpa::light_block_import::<_, _, _, RuntimeApi>(
+			let grandpa_block_import = sc_finality_grandpa::light_block_import(
```

Comment:
@philipstanislaus  I dont think this change is significant, but couldnt compile otherwise. Can you verify is ok?

Reply:
That's fine. The bracket stuff were just type hints for the compiler.